ParkingZoneApp.Tests/ModelValidationTests/EditVMTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿using ParkingZoneApp.Models;
+using ParkingZoneApp.ViewModels.ParkingZones;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ParkingZoneApp.Tests.ModelValidationTests
+{
+    public class EditVMTests
+    {
+        public static IEnumerable<object[]> testEditVMData =>
+           new List<object[]>
+           {
+                new object[] { Guid.NewGuid(), null, "Chilonzor", new DateOnly(2024, 4, 12) },
+                new object[] { null, "7Parking", "Qoraqamish", new DateOnly(2024, 4, 12) },
+                new object[] { Guid.NewGuid(), "Sharafshon", null, new DateOnly(2024, 4, 12) },
+                new object[] { Guid.NewGuid(), "Sharafshon", "Andijon", null }
+           };
+
+        [Theory]
+        [MemberData(nameof(testEditVMData))]
+        public void CreateVM_WithValidData_ShouldPassValidation(Guid? id, string name, string address, DateOnly? createdDate)
+        {
+            //Arrange
+            ParkingZone parkingZone = new ParkingZone()
```

Comment:
why did you created new parking zone and assigned to EditVM instead of just creating EditVM

Reply:
![image](https://github.com/dilmurod-rozikov/ParkingZone/assets/141717735/b24e0748-3b00-4182-8e51-f27e258c9586)
cuz I have to pass parkingZone object to the constructor of editVM